code/src/Album.css

```diff
@@ -0,0 +1,60 @@
+/* STYLING FOR ALBUM IMAGES */
+
+.Img-wrapper:hover .Icon-wrapper {
+  width: 300px;
+  height: 300px;
+  display: flex;
+  justify-content: space-evenly;
+  position: absolute;
+  background: black;
+  opacity: 0.75;
+}
+
+.Img-wrapper {
```

Comment:
How come these to starts with big letters when the rest is starting with a small letter? :)

Reply:
I believe it was a rookie mistake on this one - would you say that small letters are best to work with using when declaring classes?